# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/mochitest/test_bug1681664.html
# Commit: ced517f952ce
# Full Hash: ced517f952ce8591d012ba92711859ef89a900f7
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2021-04-15 21:46:43
# Description:
#   Bug 1704561 - Ensure that Module return value is an object before binding it to evaluationPromise; r=evilpie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D111849
# ==============================================================================

diff -r b099dbba4587 -r ced517f952ce js/xpconnect/tests/mochitest/test_bug1681664.html
--- a/js/xpconnect/tests/mochitest/test_bug1681664.html	Thu Apr 15 12:11:02 2021 +0000
+++ b/js/xpconnect/tests/mochitest/test_bug1681664.html	Thu Apr 15 12:28:48 2021 +0000
@@ -20,7 +20,7 @@
 
         Services.obs.addObserver(observer, "process-hang-report");
         try {
-          await import("test_bug1681664.js");
+          await import("test_bug1681664_helper.js");
           result.textContent = "FAIL";
         } catch (ex) {
           result.textContent = "PASS";
